### PR TITLE
Add metrics-server docs

### PR DESCRIPTION
**Pull request by Poor12**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind documentation

**What this PR does / why we need it**:

Add kubectl logs & metrics-server deployment docs

Fixes #

**Special notes for your reviewer**:

```release-note

```

### CODE REVIEW

**Comment by daixiang0** [comment]
something duplicate with #1914

**Comment by Poor12** [comment]
Waiting for 1914 merge first , then resolve the conflict.

**Comment by Poor12** [comment]
I am not sure about it.  I think It is easier for users to attach to the document.  I can also briefly explain the precautions for deploying documents.  You decide it. @fisherxu

**Comment by daixiang0** [comment]
Metrics-server is an optional choice, it is only for users who want to collect k8s metrics.

I think it would be better to provide a link about how to deploy it and comment some custom settings.

**Comment by Poor12** [comment]
Docs is improved.

File: docs/setup/keadm.md [hunk 1]
```diff
@@ -138,6 +138,28 @@ KubeEdge edgecore is running, For logs visit:  /var/log/kubeedge/edgecore.log
 
 6. Restart all the cloudcore and edgecore.
 
+### Support Metrics-server in Cloud
+1. The realization of this function point reuses cloudstream and edgestream modules. So you also need to perform all steps of *Enable `kubectl logs` Feature*.
+
+2. Since the kubelet ports of edge nodes and cloud nodes are not the same, the current release version of metrics-server does not support automatic port identification, so you need to manually compile the image yourself.
```
Comment (kevin-wangzefeng) [inline docs/setup/keadm.md:144]: > the current release version of metrics-server does not support automatic port identification

Better to point out exact version numbers of metrics-server that support and doesn't support

Comment (Poor12) [reply docs/setup/keadm.md]: Updated. Now the metrics-server is 0.3.7. This feature is estimated to be launched in 0.4.0. Some features of 0.4.0 are still in the early stages of development.

File: docs/setup/keadm.md [hunk 2]
```diff
@@ -138,6 +138,28 @@ KubeEdge edgecore is running, For logs visit:  /var/log/kubeedge/edgecore.log
...
+```
+
+3. Apply the deployment yaml. For specific deployment documents, you can refer to https://github.com/kubernetes-sigs/metrics-server/tree/master/manifests.
+
+**IMPORTANT NOTE:**
+1. Metrics-server needs to use hostnetwork network mode.
+
+2. Use the image compiled by yourself and set imagePullPolicy to IfNotPresent.
+
+3. Turn on the feature of --kubelet-use-node-status-port
```
Comment (fisherxu) [inline docs/setup/keadm.md:161]:
```suggestion
3. Enable the feature of --kubelet-use-node-status-port for Metrics-server
```
It shoulod be for Metrics-server here I think :)

**Pull request merged**